week_3_alt_assignment/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "node_express_example_template",
+  "version": "1.0.0",
+  "description": "Boilerplate for a node & express example",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jdevries3133/node_express_example_template.git"
+  },
+  "author": "Jack DeVries <jdevries3133@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jdevries3133/node_express_example_template/issues"
+  },
+  "homepage": "https://github.com/jdevries3133/node_express_example_template#readme",
+  "dependencies": {
+    "@types/node": "^16.18.14",
```

Comment:
Not sure if we want this guy

Looks like I was mistaken, `@types/express` is not here; maybe newer express doesn't have a type-stub library? I'm not sure.

Reply:
Is this what you're meaning? https://www.npmjs.com/package/@types/express

Reply:
Hm, I think I just got mixed up here. I thought I did `npm i @types/express` and nothing was installed, so I assumed that latest versions of express don't have a type-stubs library and I just didn't notice that the install failed.

Then, I just ran `npm i @types/express` and it worked! So, presumably, I never actually installed them in the first place, and that's why they weren't there............. that'll do it.

Anyway, the basic question remains: do we want to include `@types` library in course content? Will it help students or harm them?

Reply:
Personally, I don't feel too strongly about it. I think if adding them helps enough in VSCode for IDE autocomletion or hovering over things then that's cool. As long as they don't need to do anything apart from just installing the types.